examples/ASDKgram/Sample/PhotoFeedNodeController.m

```diff
@@ -93,30 +93,30 @@ - (void)loadPageWithContext:(ASBatchContext *)context
   [_photoFeed requestPageWithCompletionBlock:^(NSArray *newPhotos){
     
     [self insertNewRowsInTableView:newPhotos];
-    [self requestCommentsForPhotos:newPhotos];
+//    [self requestCommentsForPhotos:newPhotos];
```

Comment:
We should also comment at these two locations to indicate that this is not an ideal implementation, and highlight the fact that it is not trivial to do better here. In this case, this is a one-shot call for all of the photos that have just loaded, which can just the network and slows down image  loading . The only easy alternative is to perform the load when  A cell becomes visible, which results in poor user perceive performance of  comment loading  and can cause jumpy and flashy transitions as the comments always change the height of the cells once they are visible.